.github/ISSUE_TEMPLATE/feature-improvement-request.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature/Improvement request
 about: Suggest an enhancement
-title: ''
+title: '[FEATURE NAME]'
```

Comment:
I don't know why it pushed these changes. You may have to revoke these in particular